packages/site/src/routes/[dictionaryId]/contributors/+page.svelte

```diff
@@ -1,24 +1,46 @@
 <script lang="ts">
   import { t } from 'svelte-i18n';
-  import { add, deleteDocumentOnline, updateOnline, Collection } from 'sveltefirets';
+  import { add, deleteDocumentOnline, updateOnline, Collection, docStore } from 'sveltefirets';
   import { where } from 'firebase/firestore';
   import { isManager, isContributor, dictionary, admin } from '$lib/stores';
-  import type { IInvite, IHelper } from '@living-dictionaries/types';
+  import type { IInvite, IHelper, IUser } from '@living-dictionaries/types';
   import { Button, ShowHide } from 'svelte-pieces';
   import { inviteHelper } from '$lib/helpers/inviteHelper';
   import { removeDictionaryContributor } from '$lib/helpers/dictionariesManaging';
   import ContributorInvitationStatus from '$lib/components/contributors/ContributorInvitationStatus.svelte';
   import Citation from './Citation.svelte';
   import SeoMetaTags from '$lib/components/SeoMetaTags.svelte';
+  import type { Address } from '../../api/email/send/mail-channels.interface';
+  import type { Writable } from 'svelte/store';
 
   let helperType: IHelper[];
   let inviteType: IInvite[];
+  let userType: IUser;
+  let managers:IHelper[] = []
+
+  let users:Partial<Writable<IUser>>[]; // Is this ok?
+  let managerAddresses:Address[] = []
+  $: if (managers) {
+    users = managers.map(manager => {
+      const userStore = docStore<IUser>(`users/${manager.id}`, { startWith: userType, log: true });
```

Comment:
You just need a one time call here, so getDocument or getCollection is what you're looking for. Let's work on this PR together, @Danble